workflows/cloudify_system_workflows/snapshots/postgres.py

```diff
@@ -118,6 +125,7 @@ def drop_db(self):
         drop_db_bin = os.path.join(self._bin_dir, 'dropdb')
         command = [drop_db_bin,
                    '--host', self._host,
+                   '--port', self._port,
```

Comment:
fixed by changing POSTGRESQL_DEFAULT_PORT to str, because otherwise it's substr of self._host..